### ISSUE TITLE
adbus.sdbus.SdbusError: Failed to enter array b'y'

### ISSUE THREAD

**Issue by Kaveshnikov** [ccxtechnologies/adbus#40]
```python
import asyncio
import adbus

async def create_service_proxy():
    name = 'cron'
    proxy = adbus.client.Proxy(
        adbus.Service(bus='session'),
        'org.freedesktop.systemd1',
        '/org/freedesktop/systemd1/unit/%s_2eservice' % name,
        interface='org.freedesktop.systemd1.Unit',
    )
    await proxy.update()
    return proxy

def main():
    loop = asyncio.get_event_loop()
    loop.run_until_complete(create_service_proxy())
    loop.close()

if __name__ == '__main__':
    main()
```
This program fails with error:
```
File "adbus/sdbus/call.pyx", line 13, in adbus.sdbus.call_callback
  File "adbus/sdbus/message.pyx", line 219, in adbus.sdbus.Message.read
  File "adbus/sdbus/message.pyx", line 133, in adbus.sdbus.Message._read_array
  File "adbus/sdbus/message.pyx", line 228, in adbus.sdbus.Message.read
  File "adbus/sdbus/message.pyx", line 198, in adbus.sdbus.Message._read_dict
  File "adbus/sdbus/message.pyx", line 222, in adbus.sdbus.Message.read
  File "adbus/sdbus/message.pyx", line 160, in adbus.sdbus.Message._read_variant
  File "adbus/sdbus/message.pyx", line 219, in adbus.sdbus.Message.read
  File "adbus/sdbus/message.pyx", line 133, in adbus.sdbus.Message._read_array
  File "adbus/sdbus/message.pyx", line 219, in adbus.sdbus.Message.read
  File "adbus/sdbus/message.pyx", line 129, in adbus.sdbus.Message._read_array
adbus.sdbus.SdbusError: Failed to enter array b'y'
sys:1: RuntimeWarning: coroutine 'get_all' was never awaited
```
Systemd: 237
libsystemd:  237-3ubuntu10.13
adbus: 0.2.0 from pypi
python: 3.6.7

**Comment by charleseidsness** [comment]
We haven't had a chance to update the PyPi version in some time, I recommend using the GitHub version instead.

**Comment by Kaveshnikov** [comment]
I tried version from master and it works for me, but I had to revert commit ecb10a1622c765e0e85ef3aba2e4730193c3227f. What is it made for?

**Comment by charleseidsness** [comment]
Released version 1.0.1 which pulls in the latest updates, but still required 3.7. You can use it with an older version if you'd like but it isn't officially supported.
